index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet"> 
+  <script src="https://kit.fontawesome.com/9d5cd28617.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PoPraX</title>
+</head>
+<body>
+  <main>
+    <div class="top-top-nav desktop">
+      <ul>
+        <li><i class="social-media-icons fa-brands fa-twitter"></i></li>
+        <li><i class="social-media-icons fa-brands fa-facebook-f"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <nav class="top-nav desktop">
+      <a href="#home" class="logo">PoPraX</a>
+      <ul>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#top-songs">Top songs</a></li>
+        <li><a href="#featured-performers">Performers</a></li>
+        <li><a href="#home">PoPraX</a></li>
+      </ul>
+    </nav>
+    <header id="home">
+      <i class="fa-solid fa-bars menu-icon mobile"></i>
+      <div class="heading-content">
+        <h1>The cheapest tickets on the internet, period.</h1>
+      <p>Zero hidden service fees, price match guarantee, group discounts, and more. Beacuse it <b>Should</b> be this easy</p>
+      <a class="events-btn" href="#">Checkout upcoming events </a>
+      <div id="side-menu" class="menu-items">
+        <i class="fa-solid fa-xmark closebtn close-menu mobile"></i>
+        <div class="overlay-content">
+          <a href="about.html" class="close-menu">About</a>
+          <a href="#top-songs" class="close-menu">Top Songs</a>
+          <a href="#featured-performers" class="close-menu">Performers</a>
+          <a href="#partners" class="close-menu">Partners</a>
+        </div>
+      </div>
```

Comment:
- [ ] `required`
- [ ] amazing job so far 👏 👏 
- [ ] I would suggest you follow the original template according to the project the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md).
- [ ] The pages should look `almost identical` to the original design. Small adjustments like text or image changes are acceptable.
- [ ] You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
* Colors.
* Typography: font face, size and weight.
* Layout: composition and space between elements.
- [ ] please see screenshots below for comparisons

`your template`
![his template](https://user-images.githubusercontent.com/66230306/181392971-91498ffa-9edc-432f-97a5-78798d883fc8.png)

`figma template`
![header](https://user-images.githubusercontent.com/66230306/181393035-c6e28540-b73d-46a5-8f93-54a4a50c4e78.png)